src/datadog.ts

```diff
@@ -90,6 +103,14 @@ export class Datadog extends cdk.Construct {
       this.transport.applyEnvVars(lambdaFunctions);
     }
   }
+
+  public addForwarderToLogGroups(logGroups: logs.LogGroup[]) {
+    if (this.props.forwarderArn !== undefined) {
+      addForwarderToLogGroup(this.scope, logGroups, this.props.forwarderArn);
+    } else {
+      log.debug("Forwarder ARN not provided, no log group subscriptions will be added");
```

Comment:
Nit: This debug statement is exactly the same as the one in the `addLambdaFunctions` function [here](https://github.com/DataDog/datadog-cdk-constructs/blob/d5b9c0adecc1d3b165da3880fa517c5d6b4e63ca/src/datadog.ts#L98), it might be useful to keep the core message the same but differ them slightly so that you can distinguish what code path they came from if you were to look at the debug logs.